containers/hugging-face-inference/terraform/terraform.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
```

Comment:
**issue(scripting):** These kinds of commands should be managed in a `Makefile`.